TestModule/TestFunctions.cpp

```diff
@@ -4,13 +4,10 @@
 #define WIN32_LEAN_AND_MEAN
 #include <windows.h>
 #include <cstddef>
-#include <filesystem>
-
-namespace fs = std::experimental::filesystem;
 
 std::string GetOP2IniPath();
 std::string GetGameDirStdString();
-std::size_t CountVolFilesInGameDirectory();
+std::size_t CountFilesByTypeInDirectory(const std::string& directory, std::string extension);
 
```

Comment:
Does it make sense to create a header file for this function? I'm not used to seeing lone .cpp files.

I guess this is considered ok as long as we do not use the function CountFilesByTypeInDirectory outside of this file.

Reply:
Yes. I was thinking of creating a header file, though I was being lazy. I mean, common, it's just one method! :wink: 

And you know, you already had a bunch of forward declares at the top of that file. One of which being the old function being replaced. Though I'm just realizing now the forward declares are all for methods within that source file.

I should update this.